src/lgr_advanced/lgr_editor/views/create.py

```diff
@@ -131,9 +132,10 @@ def _handle_lgr_file(self, lgr_file, validating_repertoire, is_set, lgr_info_set
                                                      "Please rename it before importing it."))
 
         try:
+            self.session.save_lgr(validating_repertoire.to_lgr_info(), validating_repertoire.name)
             lgr_info = self.session.open_lgr(lgr_id,
                                              lgr_file.read(),
-                                             validating_repertoire=validating_repertoire,
+                                             validating_repertoire=validating_repertoire.name,
```

Comment:
why not put a pk here? this would need to be changed when other validating repertoire types would be created but at least we do not have to save a validating repertoire in session before

Reply:
Setting a pk does not work there since the name is the identifier used elsewhere when copying.